Program.cs

```diff
@@ -55,6 +59,7 @@ public static void AddPerson()
             people.Add(newPerson);
         }
 
+        // changing information about a person on the book
         public static void UpdatePerson()
         {
             PersonList();
```

Comment:
No need to enter the loop when the list is empty.

Reply:
I think you shouldn't use any loop here. you need to find the user who has the same id as entered. and then if there is no user you can show a message, if you find the user, you can continue to ask new name etc.